Sources/CardPayments/Models/CardVaultRequest.swift

```diff
@@ -0,0 +1,12 @@
+import Foundation
+
+public struct CardVaultRequest {
+    
+    public let card: Card
+    public let setupToken: String
```

Comment:
See comment on aligning property name with Android - https://github.com/paypal/paypal-android/pull/173/files#r1289166457

Reply:
I think in iOS, we have a pattern of using capital letters, "ID" at end of variables ending with` id`. `setupTokenID` as opposed to `setupTokenId`

Reply:
Addressed in 6d3f15c04e1b44ab49f8dcc1575456180d2dbc69